addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -92,6 +98,40 @@ func _can_drop_data(at_position: Vector2, data: Variant) -> bool:
 
 
 func _drop_data(at_position: Vector2, data: Variant) -> void:
+	var type = data.get("type", "")
+	if type == "nodes":
+		_drop_node(at_position, data)
+	elif type == "obj_property":
+		_drop_obj_property(at_position, data)
+
+
+func _drop_obj_property(at_position: Vector2, data: Variant) -> void:
+	var obj_name = str(data.get("object", "")).get_slice(":", 0)
+	var prop_name = data.get("property", "")
+	var val = data.get("value", null)
+	if obj_name == "" || prop_name == "" || val == null:
+		return
+
+	# Prepare a block to get the property's value
+	var block_definition = (
+		BlockDefinition
+		. new(
+			&"%s_get_%s" % [obj_name, prop_name],
```

Comment:
How about a setter?

Reply:
I loosely remember Orchestrator (or was Godot 3 visual scripting?) using a modifier key while dropping to generate a setter. I don't remember if Shift or Ctrl.

Reply:
Maybe we could add the property get & setter to the Variables section of the picker when it has been dragged from the inspector?

Reply:
> Maybe we could add the property get & setter to the Variables section of the picker when it has been dragged from the inspector?

This is worth a new feature ticket.

Reply:
Both:
> I loosely remember Orchestrator (or was Godot 3 visual scripting?) using a modifier key while dropping to generate a setter. I don't remember if Shift or Ctrl.

and

> Maybe we could add the property get & setter to the Variables section of the picker when it has been dragged from the inspector?

acknowledge https://github.com/endlessm/godot-block-coding/pull/296#discussion_r1822518307!

Reply:
After think twice, using a modifier key while dropping to generate a setter mentioned in https://github.com/endlessm/godot-block-coding/pull/296#discussion_r1822525086 is a good, simple and quick implementation for now.